OpenDreamRuntime/Objects/Types/DreamObjectImage.cs

```diff
@@ -37,14 +37,14 @@ public sealed class DreamObjectImage : DreamObject {
 
     public override void Initialize(DreamProcArguments args) {
         base.Initialize(args);
+        Appearance = AtomManager.GetAppearanceFromDefinition(ObjectDefinition);
```

Comment:
Would be good to avoid the `AtomManager.GetAppearanceFromDefinition()` if possible. Wouldn't we get the same behavior if the below `if` was kept the same but `Appearance = new IconAppearance()` was replaced with this?

Reply:
Any particular reason to avoid `AtomManager.GetAppearanceFromDefinition()`? It seems like the only sensible thing to do would be to duplicate its code anyway.

Reply:
I'm not saying to avoid it entirely, but only use it if there was no argument given to copy the appearance from.

Appearance creation is slow/heavy so it's good to avoid it if it's never used. Even if it's cached that's an extra unnecessary dictionary hit in what I imagine to be somewhat hot code.